vdirsyncer/cli/config.py

```diff
@@ -87,9 +88,14 @@ def _validate_collections_param(collections):
             if collection_name in collection_names:
                 raise ValueError("Duplicate value.")
             collection_names.add(collection_name)
```

Comment:
If a name starts with `!`, you're also adding it to the `collection_names` list here. You probably want to move this line to an `else` block after `if collection[0] == "!":`.

Reply:
Yes I saw that, but does it matter at this point, as `collection_names` is temporary here and as I understand only used to weed out duplicate values from the collections list. When I leave the `excludes` within the `collection_names` I don't have to bother further with that. Or did I missed something there and `collection_names` is used for anything else?

I replaced `blacklist` with `excludes`, comes in the next push